heareval/predictions/task_predictions.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, nfeatures: int, nlabels: int, prediction_type: str):
             raise ValueError(f"Unknown prediction_type {prediction_type}")
 
     def forward(self, x: torch.Tensor):
+        x, y = x
```

Comment:
What's `y` for?

Reply:
Wait, so is the x that is passed in here a tuple that contains the embeddings and the ground truth? That's quite confusing, can we split that out before this?

Reply:
I fix this in a later PR

Reply:
You can see now this is fixed